plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js

```diff
@@ -0,0 +1,7 @@
lygon=n.polygon(e,s(s({},o),{weight:0})),r}return i(e,t),e.prototype.onAdd=function(t){return this.polyline.addTo(t),this.polygon.addTo(t),this},e.prototype.onRemove=function(){return this.polyline.remove(),this.polygon.remove(),this},e.prototype.addLatLng=function(t){return this.polyline.addLatLng(t),this.polygon.addLatLng(t),this},e.prototype.getLatLngs=function(){return this.polygon.getLatLngs()[0]},e.prototype.toGeoJSON=function(){return this.polygon.toGeoJSON()},e}(n.Layer),u=function(t){for(var e=null,o=null,n=null,r=null,i=0;i<t.length;i++)for(var s=t[i],a=0;a<s.length;a++){var l=s[a],u=l[0],c=l[1];(null===e||u<e)&&(e=u),(null===o||u>o)&&(o=u),(null===n||c<n)&&(n=c),(null===r||c>r)&&(r=c)}return[e,n,o,r]},c=function(t){for(var e=null,o=null,n=null,r=null,i=0;i<t.length;i++){var s=t[i],a=s[0],l=s[1];(null===e||a<e)&&(e=a),(null===o||a>o)&&(o=a),(null===n||l<n)&&(n=l),(null===r||l>r)&&(r=l)}return[e,n,o,r]},p=function(t){for(var e=[],o=t.features.length-1;o>=0;o--){var n=d(t.features[o].geometry);e.push([n[0],n[1]]),e.push([n[2],n[3]])}return c(e)},f=function(t){for(var e=[],o=t.geometries.length-1;o>=0;o--){var n=d(t.geometries[o]);e.push([n[0],n[1]]),e.push([n[2],n[3]])}return c(e)},d=function t(e){if(e.type)switch(e.type){case"Point":return[e.coordinates[0],e.coordinates[1],e.coordinates[0],e.coordinates[1]];case"MultiPoint":case"LineString":return c(e.coordinates);case"MultiLineString":case"Polygon":return u(e.coordinates);case"MultiPolygon":return function(t){for(var e=null,o=null,n=null,r=null,i=0;i<t.length;i++)for(var s=t[i],a=0;a<s.length;a++)for(var l=s[a],u=0;u<l.length;u++){var c=l[u],p=c[0],f=c[1];(null===e||p<e)&&(e=p),(null===o||p>o)&&(o=p),(null===n||f<n)&&(n=f),(null===r||f>r)&&(r=f)}return[e,n,o,r]}(e.coordinates);case"Feature":return e.geometry?t(e.geometry):null;case"FeatureCollection":return p(e);case"GeometryCollection":return f(e);default:throw new Error("Unknown type: "+e.type)}return null},h=function(t){return!isNaN(parseFloat(t))&&isFinite(t)},g=function(t,e,o,n){var r=(n[0]-o[0])*(t[1]-o[1])-(n[1]-o[1])*(t[0]-o[0]),i=(e[0]-t[0])*(t[1]-o[1])-(e[1]-t[1])*(t[0]-o[0]),s=(n[1]-o[1])*(e[0]-t[0])-(n[0]-o[0])*(e[1]-t[1]);if(0!==s){var a=r/s,l=i/s;if(a>=0&&a<=1&&l>=0&&l<=1)return!0}return!1},y=function t(e,o){if(h(e[0][0])){if(h(o[0][0])){for(var n=0;n<e.length-1;n++)for(var r=0;r<o.length-1;r++)if(g(e[n],e[n+1],o[r],o[r+1]))return!0}else for(var i=0;i<o.length;i++)if(t(e,o[i]))return!0}else for(var s=0;s<e.length;s++)if(t(e[s],o))return!0;return!1},v=function(t,e){for(var o=!1,n=-1,r=t.length,i=r-1;++n<r;i=n)(t[n][1]<=e[1]&&e[1]<t[i][1]||t[i][1]<=e[1]&&e[1]<t[n][1])&&e[0]<(t[i][0]-t[n][0])*(e[1]-t[n][1])/(t[i][1]-t[n][1])+t[n][0]&&(o=!o);return o},m=function(t,e){for(var o=0;o<t.length;o++)if(t[o]!==e[o])return!1;return!0},M=function(t,e){return t[0]>e[0]?-1:t[0]<e[0]?1:t[1]>e[1]?-1:t[1]<e[1]?1:0},L=function(t){for(var e=[],o=0;o<t.length;o++){var n=t[o].slice();!1===m(n[0],n[n.length-1])&&n.push(n[0]),e.push(n)}return e};
```

Comment:
Can it be formatted?

Reply:
No, it's minimized for a reason.

Reply:
sad

Reply:
It's just a dependency. Neither you nor I am supposed to really care about how pretty it is. Minimization reduces size, and this is what actually matters.

Reply:
Ok. And what about "wkx.js"? Why you deleted it?

Reply:
We were bundling both unminimized and minimized versions of the same library. Please see individual commits,